sample/config/etcd.yml

```diff
@@ -39,11 +39,11 @@ cors:
 
 # List of this member's peer URLs to advertise to the rest of the cluster.
 # The URLs needed to be a comma-separated list.
-initial-advertise-peer-urls: http://localhost:2380
+initial-advertise-peer-urls: http://0.0.0.0:2380
 
 # List of this member's client URLs to advertise to the public.
 # The URLs needed to be a comma-separated list.
-advertise-client-urls: http://localhost:2379
+advertise-client-urls: http://0.0.0.0:2379
```

Comment:
These advertise urls will be a problem to use 0.0.0.0 or 127.0.0.1 when we deploy 3 etcd nodes. Advertise address is what we expose to outside world, which should be its domain address.

But for now it ok to me, you can leave a to-do comment

Reply:
merge this pr first, better to optimize the etcd config in another PR.